src/Microsoft.Health.Dicom.SqlServer/Features/ExtendedQueryTag/SqlExtendedQueryTagStoreV36.cs

```diff
@@ -0,0 +1,70 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
+using Microsoft.Health.Dicom.SqlServer.Extensions;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema.Model;
+using Microsoft.Health.SqlServer.Features.Client;
+using Microsoft.Health.SqlServer.Features.Storage;
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.ExtendedQueryTag;
+
+internal class SqlExtendedQueryTagStoreV36 : SqlExtendedQueryTagStoreV16
+{
+    public SqlExtendedQueryTagStoreV36(SqlConnectionWrapperFactory sqlConnectionWrapperFactory, ILogger<SqlExtendedQueryTagStoreV36> logger)
+        : base(sqlConnectionWrapperFactory, logger)
+    {
+    }
+
+    public override SchemaVersion Version => SchemaVersion.V36;
+
+    public override async Task<IReadOnlyList<ExtendedQueryTagStoreJoinEntry>> GetExtendedQueryTagsAsync(int limit, long offset = 0, CancellationToken cancellationToken = default)
```

Comment:
same here

Reply:
Same change from `INT` to `BIGINT`:
<img width="1816" alt="image" src="https://github.com/microsoft/dicom-server/assets/2955179/1126bbb1-e7d1-4856-9ed4-6126b066e879">